src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/vsts_cd_provider.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+from sys import stderr
+from vsts_cd_manager.continuous_delivery_manager import ContinuousDeliveryManager
+from azure.cli.core._profile import Profile
+
+
+class VstsContinuousDeliveryProvider(object):
+    def __init__(self):
+        self._progress_last_message = ''
+
+    # pylint: too-many-arguments
```

Comment:
Remove this line. You have it covered on the next line and the statement is actually wrong.